s3_bucket_util_test.go

```diff
@@ -589,6 +589,29 @@ func getS3LargeBucketTests(ctx context.Context, tempdir, s3BucketName, s3Prefix,
 	}
 }
 
+// TODO: write archive-specific tests for push/pull.
```

Comment:
are you going to do this here? or in another ticket/PR?

Reply:
I actually already wrote them below in `TestS3ArchiveBucket`. I removed these lines.